lua/complete/init.lua

```diff
@@ -0,0 +1,12 @@
+-- Use <Tab> and <S-Tab> to navigate through popup menu
```

Comment:
What does this do?

Reply:
Crap.  That shouldn't be in there.  I guess I missed this when I was resolving a merge conflict.  I started to experiment with replacing compe with completion-nvim to get emmet stuff working for myself.  It shouldn't be in this pull.  lua/complete/init.lua  can be safely deleted.  The code you see in there is from the example setup on the completion-nvim readme.